### ISSUE TITLE
[Bug] Sync cards switch On and OFF from Credit cards is not announced 

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#19564]
## Steps to reproduce
1. Enable talkback.
2. Reach the credit cards section and select `Sync cards`.
3. Turn the Sync cards On or OFF.
### Expected behavior
Turning the Sync card On or OFF should be announced.
### Actual behavior
Sync cards switch On and OFF from Credit cards is not announced 
### Device information

* Device: **Google Pixel 4 XL (Android 11).**
* Firefox for Android version: **Nightly 5/13.**

https://user-images.githubusercontent.com/42831109/118819545-1cb73d80-b8be-11eb-9fab-9796abefde77.mp4

**Comment by mcarare** [comment]
From my testing, this problem is specific only to Android 11 devices.

**Comment by mcarare** [comment]
Note for QA: As we use a SwitchCompat for that setting Android 11 will announce the value change as it does for checkboxes : checked / not checked. While it might seem confusing, the announcement is relevant to the Talkback user on Android 11 platform as it is the default one for two states buttons.

**Comment by abodea** [comment]
Verified as fixed on the latest **Nightly 6/2 and Beta 90** with **Google Pixel 4 XL (Android 11).**
